dist/ExpenseManager.js

```diff
@@ -0,0 +1,40 @@
+class ExpenseManager {
+    constructor() {
+        this.expenses = []
+        this.total = []
+    }
+
+    addExpense(expense) {
+        $.post('/expense', expense, function(result) {
+            console.log(result)
+        })
+    }
+    
+    getExpense(group, startDate = 0, endDate = 0) {
+        return new Promise(resolve => {
```

Comment:
instead of returning a new Promise (which is fine), you could make this method an `async` method (and `async` methods/functions returns a promise by default)